AppCenterDistribute/AppCenterDistribute/MSDistributeUtil.m

```diff
@@ -122,4 +121,19 @@ NSComparisonResult MSCompareCurrentReleaseWithRelease(MSReleaseDetails *releaseB
 
 @implementation MSDistributeUtil
 
++ (MSUpdateTrack)storedUpdateTrack {
+  NSNumber *updateTrack = [MS_USER_DEFAULTS objectForKey:kMSDistributionUpdateTrackKey];
+  if (updateTrack) {
+    if ([self isValidUpdateTrack:[updateTrack intValue]]) {
+      return [updateTrack intValue];
+    }
+    MSLogWarning([MSDistribute logTag], @"Corrupted stored update track, switching to public.");
```

Comment:
this warning appears on first launch when there is no data at all.
We should not print this warning when no data, only when corrupted stored data like android.